compiler/control/OMROptions.cpp

```diff
@@ -5182,12 +5183,6 @@ bool OMR::Options::showOptionsInEffect()
 
 bool OMR::Options::fePreProcess(void*)
    {
-#if !defined(J9_PROJECT_SPECIFIC)
-   TR::Compiler->host.setNumberOfProcessors(2);
-   TR::Compiler->target.setNumberOfProcessors(2);
-   TR::Compiler->host.setSMP(true);
```

Comment:
As noted above, in the end we want SMP flag to be `true`

Reply:
SMP is initialized to `true` in `initializeTargetEnvironment`/`initializeRelocatableTargetEnvironment()` so I removed the code here.